documentation/commands/git-new-pull-request.md

```diff
@@ -12,5 +12,5 @@ git new-pull-request
 
 #### DESCRIPTION
 
-Opens a browser to a new pull request for the current branch.
+Opens a browser window to the "new pull request" page for the current branch on the code hosting platform.
```

Comment:
I like the new phrasing!

I have added a sentence that explains that the PR only shows changes against the immediate parent branch. What do you think about this?